### PR TITLE
build apps with engine provided frontend snapshot

### PR DESCRIPTION
**Pull request by andryblack**
Resolve https://github.com/meta-flutter/meta-flutter/issues/575

### CODE REVIEW

**Review by jwinarske — CHANGES_REQUESTED** [review]
other than requested change LTGM

File: conf/include/common.inc [hunk 1]
```diff
@@ -477,13 +477,24 @@ def build_app(d, source_dir, flutter_sdk, pubspec_appname, flutter_application_p
...
+            shutil.rmtree(f'{source_dir}/engine_sdk', ignore_errors=True)
+
+            staging_datadir = d.getVar('STAGING_DATADIR')
+            cmd = f'unzip {staging_datadir}/flutter/{flutter_sdk_version}/{runtime_mode}/engine_sdk.zip \
+                -d {source_dir}/engine_sdk'
+            run_command(d, cmd, source_root, env)
+
             if not new_build_scheme:
                 dart_runtime = f'{flutter_sdk}/bin/cache/dart-sdk/bin/dart'
                 frontend_snapshot = f'{flutter_sdk}/bin/cache/artifacts/engine/linux-{clang_build_arch(d)}/frontend_server.dart.snapshot'
```
[the remark this answers is not in the document]
Comment (andryblack) [reply conf/include/common.inc:492]: need more research, some executables builded for target machine. 
Im return to it at this week

[the remark this answers is not in the document]
Comment (jwinarske) [reply conf/include/common.inc:492]: @andryblack are you still planning on working on this?

**Comment by jwinarske** [comment]
closing due to lack of response

**Pull request closed without merging**